api/mysagw/healthz/conftest.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from django.core import management
+
+
+@pytest.fixture
+def roll_back_migrations():
+    """Roll back database migrations to ensure unapplied migrations exist."""
+    # undo applied migrations for app contenttypes
+    management.call_command("migrate", "contenttypes", "zero")
+
+    yield
+
+    # re-apply migrations for app contenttypes
+    management.call_command("migrate", "contenttypes")
```

Comment:
```suggestion
    try:
        yield
    finally:
        # re-apply migrations for app contenttypes
        management.call_command("migrate", "contenttypes")
```

Otherwise, the rollback *might* not properly happen in case of exceptions

Reply:
Fixed.

This was a copy/paste from caluma: https://github.com/projectcaluma/caluma/blob/main/caluma/caluma_core/conftest.py#L355

Reply:
https://github.com/projectcaluma/caluma/pull/1840